packages/driver/src/cy/commands/request.ts

```
@@ -138,6 +139,16 @@ export default (Commands, Cypress, cy, state, config) => {
         $errUtils.throwErrByPath('request.url_wrong_type')
       }
 
+      if (options.fixture) {
+        if (options.body) {
+          $errUtils.throwErrByPath('request.body_and_fixture')
+        }
+
+        if (!userOptions.method) {
+          options.method = 'POST'
+        }
```

I wouldn't expect to cohersing this to `POST`.  Why do you believe we should? Would it be better to throw an error being explicit that this is only supported with the POST method?

---

I agree with you. This was just part of the suggested API from the start